### PR TITLE
Add quick amount buttons to forms

### PR DESCRIPTION
**Pull request by zerts**

### CODE REVIEW

**Comment by everdimension** [comment]
I think that `button`s inside a `label` do not make sense:
image

File: src/ui/pages/SendForm/fieldsets/TokenTransferInput/TokenTransferInput.tsx [hunk 1]
```diff
@@ -76,6 +82,26 @@ export function TokenTransferInput({ sendView }: { sendView: SendFormView }) {
...
       <FormFieldset
         title="Asset"
+        endTitle={
+          tokenItem && positionBalanceCommon ? (
+            <HStack gap={16} alignItems="center">
+              {QUICK_AMOUNTS.map(({ factor, title }) => (
+                <QuickAmountButton
+                  key={factor}
+                  title={title}
+                  onClick={() => {
```
Comment (everdimension) [inline src/ui/pages/SendForm/fieldsets/TokenTransferInput/TokenTransferInput.tsx:92]: I think we should focus the input when quick amount is clicked

File: src/ui/shared/forms/QuickAmounts.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import { UnstyledButton } from 'src/ui/ui-kit/UnstyledButton';
+
+export const QUICK_AMOUNTS: { title: string; factor: number }[] = [
+  { title: '25%', factor: 0.25 },
+  { title: '50%', factor: 0.5 },
+  { title: 'MAX', factor: 1 },
+];
+
+export function QuickAmountButton({
+  title,
```
Comment (everdimension) [inline src/ui/shared/forms/QuickAmounts.tsx:12]: `title` can be confused with the native `title` attribute. Why not `children`?

File: src/ui/ui-kit/FormFieldset/FormFieldset.tsx [hunk 3]
```diff
@@ -64,9 +66,16 @@ export const FormFieldset = React.forwardRef<
...
         <VStack gap={4} style={{ width: '100%' }}>
-          <UIText kind="small/regular" as="label" ref={labelRef}>
-            {title}
-          </UIText>
+          <HStack gap={8} justifyContent="space-between" alignItems="center">
+            <UIText kind="small/regular" as="label" ref={labelRef}>
+              {title}
+            </UIText>
+            {endTitle ? (
+              <UIText kind="small/regular" as="label" ref={labelRef}>
```
Comment (everdimension) [inline src/ui/ui-kit/FormFieldset/FormFieldset.tsx:74]: perhaps remove `as label`

**Review by everdimension — CHANGES_REQUESTED** [review]
See comments, all good otherwise:)

File: src/ui/ui-kit/FormFieldset/FormFieldset.tsx [hunk 4]
```diff
@@ -64,9 +66,12 @@ export const FormFieldset = React.forwardRef<
         }}
       >
         <VStack gap={4} style={{ width: '100%' }}>
-          <UIText kind="small/regular" as="label" ref={labelRef}>
-            {title}
-          </UIText>
+          <HStack gap={8} justifyContent="space-between" alignItems="center">
+            <UIText kind="small/regular" as="label" ref={labelRef}>
```
Comment (zerts) [inline src/ui/ui-kit/FormFieldset/FormFieldset.tsx:70]: Not sure if we need label here as well. However, will not change it here for now.

**Review by everdimension — APPROVED** [review]

**Pull request merged**